src/couch_epi_sup.erl

```diff
@@ -30,7 +30,13 @@
 %% ===================================================================
 
 start_link() ->
-    supervisor:start_link({local, ?MODULE}, ?MODULE, []).
+    case supervisor:start_link({local, ?MODULE}, ?MODULE, []) of
+        {ok, _Pid} = Reply ->
```

Comment:
I'd rather see this over in the _app.erl file at https://github.com/apache/couchdb-couch-epi/blob/master/src/couch_epi_app.erl#L20

This is about application startup, doing some work after the supervisor starts, rather than something about the supervisor itself.